x/sequencer/keeper/msg_server_increase_bond.go

```diff
@@ -39,9 +39,9 @@ func (k msgServer) IncreaseBond(goCtx context.Context, msg *types.MsgIncreaseBon
 	return &types.MsgIncreaseBondResponse{}, err
 }
 
-func (k msgServer) bondUpdateAllowed(ctx sdk.Context, msg *types.MsgIncreaseBond) (types.Sequencer, error) {
+func (k msgServer) bondUpdateAllowed(ctx sdk.Context, senderAddress string) (types.Sequencer, error) {
 	// check if the sequencer already exists
-	sequencer, found := k.GetSequencer(ctx, msg.Creator)
+	sequencer, found := k.GetSequencer(ctx, senderAddress)
```

Comment:
I think it would have been clearer to just get the sequencer above this method, the name `bondUpdateAllowed` does more than just a check - it doesn't simply return a boolean